odoo/addons/base/models/res_partner.py

```diff
@@ -1092,6 +1079,8 @@ def find_or_create(self, email, assert_valid_email=False):
 
     def address_get(self, adr_pref=None):
         """ Find contacts/addresses of the right type(s) by doing a depth-first-search
+        through descendants within company boundaries then continuing the search at the
+        ancestors that are within the same company boundaries.
```

Comment:
Seems somehow duplicated lines from below ? Error when rebasing probably ? As code does not seem to change (at least in this first commit), also maybe a remaining of first implementation ?